webapp/src/components/TransferAssetForm/TransferAssetForm.css

```diff
@@ -0,0 +1,13 @@
+.TransferAssetForm .ui.input.address-input {
+  max-width: 450px !important;
+  display: block;
+}
+
+.TransferAssetForm .ui.input.address-input input {
+  width: 100% !important;
```

Comment:
Are these `!important`s necessary?
even if they're copied from another .css file I think we can avoid them using specificity

Reply:
The `max-width` it is being used but we can remove both due other inputs don't have a `max-width`